CMakeLists.txt

```diff
@@ -29,27 +30,27 @@ endif(BUILD_TEST)
 # - specify -DCMAKE_C_FLAGS=... on invocation of cmake, or
 # - specify CFLAGS=... on invocation of make
 IF(CMAKE_COMPILER_IS_GNUCC)
-  SET(CMAKE_C_FLAGS_DEBUG "-g -Wall -Wextra -O0") # will be added to CMAKE_C_FLAGS when CMAKE_BUILD_TYPE is "Debug"
+	SET(CMAKE_C_FLAGS_DEBUG "-g -Wall -Wextra -O0") # will be added to CMAKE_C_FLAGS when CMAKE_BUILD_TYPE is "Debug"
```

Comment:
looks like you've reformatted unrelated lines.  should omit these.

Reply:
Yes, I got inspired, but I can remove them.